examples/identity_cuda/eval.cu

```diff
@@ -0,0 +1,138 @@
+#include <chrono>
```

Comment:
should this file be in this folder? or are you envisioning that people copy paste some version of this per kernel

Reply:
for this "demo", yes. for the reference-kernels repo, we'd have one master copy, and then symlink that into the individual task directories. If we make minor fixes, they automatically propagate to existing tasks, if we want to do breaking changes, we can make a copy and symlink that into any new task, leaving existing stuff working.